plutus-tx-plugin/test/Plugin/Profiling/addInt.plc.golden

```diff
@@ -0,0 +1,2 @@
+[ [0] entering Name {nameString = "addInt", nameUnique = Unique {unUnique = 81}}
```

Comment:
Okay, we definitely don't want to print the names like this! Use one of the pretty-printing functions

Reply:
But it looks like the entering and exiting lines are in the right order, hooray!

Reply:
OK I think the golden files and the output files of the executable look better now.